src/java.base/share/classes/java/lang/foreign/MemorySegment.java

```diff
@@ -565,9 +565,7 @@ default MemorySegment asSlice(long offset, MemoryLayout layout) {
      * @return a slice of this memory segment.
      * @throws IndexOutOfBoundsException if {@code offset < 0}, or {@code offset > byteSize()}.
      */
-    default MemorySegment asSlice(long offset) {
```

Comment:
The interface is sealed and this method is always overridden. So, no need for a default method.